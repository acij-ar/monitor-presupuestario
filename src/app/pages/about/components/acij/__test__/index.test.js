const React = require('react');
const { shallow } = require('enzyme');

const AboutACIJ = require('..');

describe('AboutACIJ component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<AboutACIJ/>);
    expect(wrapper).toMatchSnapshot();
  });
});
