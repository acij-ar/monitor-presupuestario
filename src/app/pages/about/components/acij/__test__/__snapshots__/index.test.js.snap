// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutACIJ component should match snapshot 1`] = `
<ContentSection
  id="about-acij"
  title="ACIJ"
>
  <p>
    La Asociación Civil por la Igualdad y la Justicia (ACIJ) es una organización apartidaria, sin fines de lucro, dedicada a la defensa de los derechos de los grupos más desfavorecidos de la sociedad y el fortalecimiento de la democracia en Argentina.
  </p>
  <p>
    Fundada en 2002, ACIJ tiene por objetivos defender la efectiva vigencia de la Constitución Nacional y los principios del estado de derecho, promover el cumplimiento de las leyes que protegen a los grupos desaventajados y la erradicación de toda práctica discriminatoria, así como también contribuir al desarrollo de prácticas participativas y deliberativas de la democracia.
  </p>
</ContentSection>
`;
